bedrock/firefox/templates/firefox/firstrun/new-firstrun.html

```diff
@@ -12,6 +12,8 @@
   {% javascript 'firefox_new_firstrun' %}
 {% endblock %}
 
+{% block global_nav %}{% endblock %}
+
```

Comment:
I think we're confusing things here maybe, but the bug isn't really clear. I believe the bug is saying they are showing the same FxA UI for the new experiments, so I don't think we need to touch the old template. The only ask is to change the `utm_source` from what I can tell.